src/App.js

```diff
@@ -43,6 +43,8 @@ function App() {
             path="/reviews/:reviewId"
             element={<SingleRev loading={loading} setLoading={setLoading} />}
           />
+          <Route path="/*" element={<ErrorPage />} />
+          <Route path="/WrongCat" element={<WrongCat/>}/>
```

Comment:
You shouldn't need this as you're handling the wrong category from within /categories/:category_id